src/reward_splitter/graph.py

```diff
@@ -0,0 +1,108 @@
+from collections import defaultdict
+
+from eth_typing import BlockNumber, ChecksumAddress
+from gql import gql
+from web3 import Web3
+from web3.types import Wei
+
+from src.common.clients import graph_client
+from src.reward_splitter.typings import (
+    ExitRequest,
+    RewardSplitter,
+    RewardSplitterShareHolder,
+)
+
+
+async def graph_get_reward_splitters(
+    block_number: BlockNumber, vaults: list[ChecksumAddress]
+) -> list[RewardSplitter]:
+    query = gql(
+        '''
+        query Query($block: Int, $first: Int, $skip: Int, $vaults: [String]) {
```

Comment:
The `claimer` field will be added to the `RewardSplitter` in the subgraph. Please use it in the filter, so that only the reward splitters where operator wallet equals to the `claimer` are used.